services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None
+) -> QuerySet:
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+    if genres_ids is not None and actors_ids is not None:
+        return Movie.objects.filter(
+            genres__in=genres_ids,
+            actors__in=actors_ids
+        )
+    if genres_ids is not None:
+        return Movie.objects.filter(genres__in=genres_ids)
+    if actors_ids is not None:
+        return Movie.objects.filter(actors__in=actors_ids)
```

Comment:
You can create a queryset and after that if genres_ids is not None - filter by genres, if actors_ids is not None - filter by actors and the only once return the queryset.  In this case you will have only 2 if statements instead of 4.

Reply:
fixed